flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlerParameterTest.java

```diff
@@ -115,7 +120,15 @@ static void init() throws Exception {
 				LAST_SUBMITTED_JOB_GRAPH_REFERENCE.set(jobGraph);
 				return CompletableFuture.completedFuture(Acknowledge.get());
 			})
-			.build();
+			.setRequestJobResultFunction(jobID ->
```

Comment:
Why this change?

Reply:
Removed. Leftover from previous version.